pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -307562,7 +307562,6 @@ self: {
        description = "Cabal plugin for UUAGC";
        license = lib.licenses.bsd3;
        hydraPlatforms = lib.platforms.none;
```

Comment:
If you leave `hydraPlatforms = lib.platforms.none`, then this package doesn't actually get built on Hydra.

```suggestion
```

Reply:
Thanks, done!

Reply:
@FPtje Oh, did you possibly forget to push?

Reply:
Ah, I had committed through the "Commit suggestion" button, but that sadly throws a consistent error. I've now committed and pushed it manually as 512c48d